pkgs/by-name/wi/winbox4/package.nix

```diff
@@ -0,0 +1,78 @@
+{
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  fontconfig,
+  freetype,
+  lib,
+  libGL,
+  libxkbcommon,
+  makeDesktopItem,
+  stdenvNoCC,
+  unzip,
+  xorg,
+  zlib,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "winbox";
+  version = "4.0beta3";
+
+  src = fetchurl {
+    name = "WinBox_Linux-${finalAttrs.version}.zip";
+    url = "https://download.mikrotik.com/routeros/winbox/${finalAttrs.version}/WinBox_Linux.zip";
+    hash = "sha256-GBxM6NKE05x1DZQKegJY7c5awvSenfWVgrr4Kj9esZg=";
+  };
+
+  sourceRoot = ".";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+    unzip
+  ];
+
+  buildInputs = [
+    fontconfig
+    freetype
+    libGL
+    libxkbcommon
+    xorg.libxcb
+    xorg.xcbutilimage
+    xorg.xcbutilkeysyms
+    xorg.xcbutilrenderutil
+    xorg.xcbutilwm
+    zlib
+  ];
+
+  installPhase = ''
```

Comment:
Maybe add a wrapper around the `WinBox` binary for migrating saved devices over? The announcement mentiones this script:

```shell
cp ~/.wine/drive_c/users/$USER/Application\ Data/Mikrotik/Winbox/Addresses.cdb ~/.local/share/MikroTik/WinBox
```